api/favorites/favoritesRouter.js

```diff
@@ -0,0 +1,63 @@
+const router = require('express').Router();
+
+const Favorites = require('./favoritesModel');
+// const City = require('../city/cityModel')
+// const Profile = require('../profile/profileModel')
+
```

Comment:
let's remove these variables if we don't need them right now

Reply:
ok done